pkg/controller/mustgatherservice/mustgatherservice.go

```diff
@@ -392,32 +392,37 @@ func (r *ReconcileMustGatherService) updateMustGatherServiceIngress(instance *op
 
 }
 
-func (r *ReconcileMustGatherService) desiredMustGatherServiceIngress(instance *operatorv1alpha1.MustGatherService) *extensionsv1.Ingress {
+func (r *ReconcileMustGatherService) desiredMustGatherServiceIngress(instance *operatorv1alpha1.MustGatherService) *networkingv1.Ingress {
 	appName := mustGatherResourceName
 	labels := labelsForMustGatherService(appName, instance.Name)
 	annotations := annotationsForMustGatherServiceIngress()
+	pathType := networkingv1.PathType("ImplementationSpecific")
```

Comment:
I am not super strict on that, but I'd prefer to avoid using the static strings in the function vs using the contant variables defined in the file.

Reply:
Introduced constant file to address these values.